pkgs/stdenv/generic/default.nix

```diff
@@ -153,7 +153,7 @@ let
       }) mkDerivation;
 
       # Slated for deprecation in 21.11
-      lib = if config.allowAliases then builtins.trace
+      lib = if config.allowAliases or false then builtins.trace
```

Comment:
I think showing the warning would be a better default, since this deprecation is fairly recent and not all configurations may have updated their `stdenv.lib` usage.

Reply:
Yeah, you're right. I was going with the mindset of "bools default to false", rather than thinking about the context.

https://github.com/NixOS/nixpkgs/pull/112260